### PR TITLE
Create a views folder

### PR DESCRIPTION
**Pull request by MynahMarie**
Created basic skeleton for frontend rendering of the information so that we can start implementing the search functionality

Relates #2

### CODE REVIEW

File: public/main.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+function fetchGET(url, cb) {
+    var xhr = new XMLHttpRequest();
+    xhr.onreadystatechange = function() {
+        if (xhr.readySate === 4 && xhr.status === 200) {
+            cb(null, JSON.parse(xhr.responseText));
+        } else if (xhr.readyState === 4 && xhr.status !== 200) {
+            cb(xhr.status);
+        } else {
```
Comment (sophieeejw) [inline public/main.js]: Why do we need this else block??

Comment (MynahMarie) [reply public/main.js]: It's just my personal preference. I've used this function before in many projects so I know it works, if you have a suggestion to improve it feel free!

File: package.json [hunk 2]
```diff
@@ -18,9 +18,15 @@
...
+    "morgan": "^1.9.0",
+    "nodemon": "^1.14.11"
+  },
   "dependencies": {
     "body-parser": "^1.18.2",
+    "emotional": "0.0.3",
     "express": "^4.16.2",
-    "nodemon": "^1.14.11"
+    "polarity": "^3.0.1",
+    "sentiment": "^4.1.0"
```
Comment (Karyum) [inline package.json]: whats this?

File: package.json [hunk 3]
```diff
@@ -18,9 +18,15 @@
     "url": "https://github.com/nervnews/nerv/issues"
   },
   "homepage": "https://github.com/nervnews/nerv#readme",
+  "devDependencies": {
+    "morgan": "^1.9.0",
+    "nodemon": "^1.14.11"
+  },
   "dependencies": {
     "body-parser": "^1.18.2",
+    "emotional": "0.0.3",
```
Comment (Karyum) [inline package.json]: what is this !!?

Comment (sophieeejw) [reply package.json]: module for sentiment analysis!

File: package.json (hunk 2 above)
Comment (sophieeejw) [reply package.json]: module for sentiment analysis!

File: package.json (hunk 3 above)
Comment (Karyum) [reply package.json]: https://github.com/nervnews/nerv/wiki/Sentiment-Analysis

File: package.json (hunk 2 above)
Comment (Karyum) [reply package.json]: https://github.com/nervnews/nerv/wiki/Sentiment-Analysis

**Comment by Karyum** [comment]
:shipit:

**Pull request merged**